src/components/dashboard/SendConfirmation.js

```diff
@@ -40,19 +39,19 @@ const SendConfirmation = ({ screenProps, styles }: ReceiveProps) => {
                 <Icon name="send" size={getDesignRelativeHeight(45)} color="white" />
               </View>
             </Section.Row>
-            <Section.Title fontWeight="medium" style={styles.amountWrapper}>
+            <Section.Row fontWeight="medium" style={styles.amountWrapper}>
```

Comment:
Why row instead of title?

Reply:
This resolve crash on Android, when Title wrapping an element different than a text, Android don't support